raiden/tests/fixtures/variables.py

```diff
@@ -345,6 +345,12 @@ def skip_if_not_geth(blockchain_type):
 
 
 @pytest.fixture
-def start_raiden_apps():
+def start_raiden_apps() -> bool:
     """Determines if the raiden apps created at test setup should also be started"""
     return True
+
+
+@pytest.fixture
```

Comment:
We could make such a fixture session scoped. This would make it a bit easier to move to more session scoped fixtures later on (if we ever get to it).

Reply:
Not sure about this one, because we have integration tests that need the API and most that don't need it.

Reply:
OK, it probably doesn't make much difference either way.